### PR TITLE
Fix build & console errors

### PR DESCRIPTION
**Pull request by felixmosh**
The app was missing redux & had several react errors on runtime

### CODE REVIEW

**Comment by felixmosh** [comment]
Ping?

File: samples/photo_album/src/components/PhotosUploader.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ class PhotosUploader extends Component {
                     {this.props.uploadedPhotos.map(uploadedPhoto => {
                         return (
                             <UploadedPhotoStatusContainer
-                                key={uploadedPhoto.public_id}
+                                key={uploadedPhoto.fileName}
```
Comment (maoznir) [inline samples/photo_album/src/components/PhotosUploader.js]: @felixmosh
File name might not be unique,  can you please change key to be the index of map?

Comment (felixmosh) [reply samples/photo_album/src/components/PhotosUploader.js]: To use index as react's key is considered as a bad practice, but for demo sake it enough

Comment (maoznir) [reply samples/photo_album/src/components/PhotosUploader.js]: @felixmosh merged. thanks for the pr and changes!

**Pull request merged**